app/render.js

```diff
@@ -2,20 +2,32 @@
 
 import React from 'react';
 // $FlowFixMe hydrate() is missing in flow-typed react-dom
-import { hydrate } from 'react-dom';
+import { hydrate, render } from 'react-dom';
+import { setConfig } from 'react-hot-loader';
 import { match } from 'react-router';
 import Root from './Root';
-import { AppContainer } from 'react-hot-loader';
 import routes from 'app/routes';
 import type { Store } from 'app/types';
+setConfig({
+  //pureSFC: true
+  ignoreSFC: true, // RHL will be __completely__ disabled for SFC
```

Comment:
Maybe write what RHL and SFC stands for in the comment for readability

Reply:
Nice catch! Almost forgot this..

Did some testing with this and https://github.com/mzgoddard/hard-source-webpack-plugin/. hard-source-webpack-plugin makes `yarn start` take ~7-8.sec, and I had to tweak react-hot-loader to make it work. Not 110% stable tho, more like 60-70%, so we can look into that later.

From what I remember, we may as well just remove this setConfig altogether. 

Some background info/kok
https://github.com/gaearon/react-hot-loader/issues/1088#issuecomment-452904866

Reply:
Yee. Works well without it, so I removed it now.